src/request.cpp

```diff
@@ -25,17 +25,166 @@
 #include "mega/megaclient.h"
 
 namespace mega {
+
+//ProcResultQueue::entry::~entry()
```

Comment:
We have a powerful versioning tool, no need to keep obsolete code here as comments. Please, clean it up.

Reply:
quite right, I thought I had already done that actually - maybe I had only removed it from the header.